pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -120,6 +120,30 @@ var osMap = map[string]string{
 	"windows_11":           "win11",
 }
 
+// Map of ovirt guest ids to instance type preference names.
+var osInstanceTypeMap = map[string]string{
+	"rhel_7x64":         "rhel.7",
+	"rhel_8x64":         "rhel.8",
+	"rhel_9x64":         "rhel.9",
+	"ubuntu_12_04":      "ubuntu",
+	"ubuntu_12_10":      "ubuntu",
+	"ubuntu_13_04":      "ubuntu",
+	"ubuntu_13_10":      "ubuntu",
+	"ubuntu_14_04":      "ubuntu",
+	"windows_10":        "windows.10", // windows also have virtio: e.g windows.10.virtio might be good to imports from ovirt?
```

Comment:
that would be a benefit of having this configurable - then we can default it to windows.10.virtio, and if customers have Windows guests in oVirt without virtio, then they can switch

Reply:
ok, so let's do all ovirt windows vms with virtio